int32[] ids
float64[] coverages
---
